context/design-tokens/README.md

```diff
@@ -1,157 +1,218 @@
 
----
-## Design Tokens Naming Conventions
-### Literal Design Tokens
-These are the ‘locked’ tokens, that hold a value that should not change. These are primarily used to to create the Alias Design Tokens that can be used in developing components.
-The Design values we have integrated into a Design Tokens workflow are:
-#### Colour
-The `color` Design Tokens will be used for any Alias Design Token defined.
-As the current colour palettes used are relatively small we will name most of the colours. For the grayscale colours we will apply a numerical scale to go from white to black, `100` to `900`. This naming convention allows for additional grayscale colours to be added over time if needed.
-There are four categories for the colour palettes used across Springer Nature:
-1. UI - These are used for the foreground and background colours of the web page. The page background, the header background, the footer background. As well as interactive states of text links and buttons.
-2. grayscales - These are used for border colours, text colours, and backgrounds.
-3. Information - These are used for different states like the error state of a form input, or the focus state of a text link.
-4. Brand Specific Colours - These are specific to the various brands across Springer Nature and could be used for text, border, background colour etc.
-##### Example
+To delete any existing `index.scss` files, run:
+```bash
+npm run tokens:delete-index
 ```
-{
-  "color": {
-		"ui": {
-			"universal-dark-blue": {
-				"value": "#01324b"
-			}
-		},
-		"grayscale": {
-			"100": {
-				"name": "grayscale-100",
-				"value": "#ffffff"
-  		},
-			"200": {
-				"name": "grayscale-200",
-				"value": "#f8f8f8"
-			}
-		}
-  }
-}
+To generate new `index.scss` files, run:
+```bash
+npm run tokens:build-index
 ```
-#### Typography
-There are several Literal Design Tokens for typography. They include:
-- the `font-family` which defines the preferred typeface to be used, and for the web a group of fallbacks.
-- the `font-size`. These will be defined in using the `rem` unit and will also make use of a numerical scale where `400` is the same as `1rem` or `16px`. Again, this helps to add font sizes to the scale if needed in the future.
-- the `font-weight`. These will be defined using the existing numeric keyword values
-- the `line-height`. These will be defined with unitless values and will use descriptive names like ‘tight’ and ‘loose’.
-- the `letter-spacing`. These will be defined using the `rem` unit and will use a numerical scale. As `letter-spacing` is tiny in comparison to `font-size` we use a similar scale but with `10`s prefixed with a `0`.
-##### Example
+#### Compiling design tokens for the Elements components
+
+To generate the all `scss` files for the Elements components brand components from the design tokens, run:
```

Comment:
```suggestion
To generate the all `scss` files for Elements components from the design tokens, run:
```

Reply:
Oh wait - what's the difference between this one and the instruction below? Do we mean brand-specific components here, and global below? Or something else?